index.js

```diff
@@ -0,0 +1,53 @@
+const dotenv = require('dotenv');
+const { Client, Collection, GatewayIntentBits } = require('discord.js');
+const fs = require('fs');
+const path = require('path');
+
+dotenv.config();
+
+const commandsPath = path.join(__dirname, 'commands');
+const eventsPath = path.join(__dirname, 'events');
+const customIdsPath = path.join(__dirname, 'customIds');
+
+const commandFiles = fs.readdirSync(commandsPath).filter(file => file.endsWith('.js'));
+const eventFiles = fs.readdirSync(eventsPath).filter(file => file.endsWith('.js'));
+const customIdFiles = fs.readdirSync(customIdsPath).filter(file => file.endsWith('.js'));
```

Comment:
Could we refactor this to have something like:

**index.js**
```js
const events = require('./events/index.js');

// Event handler
for (const event of events) {
	if (event.once) {
		client.once(event.name, (...args) => event.execute(...args));
	} else {
		client.on(event.name, (...args) => event.execute(...args));
	}
}
```

**/events/index.js**
```js
module.exports = [
  require('./ready.js'),
  require('./interactionCreate.js')
]
```

To me, this seems more intuitive and less of a security risk since we aren't just blindly importing files. What do you think?

Reply:
That's totally fine. The only reason I was blindly importing files is so that we wouldn't have to manually add another file to import, but this discord bot is (supposedly) small so that can work.